tests/test_codelists.py

```diff
@@ -53,7 +66,17 @@ def codelists_path(tmp_path):
     yield tmp_path
 
 
-def test_codelists_check(codelists_path):
+def test_codelists_check(mock_check, codelists_path):
+    mock_check(response={"status": "ok"})
+    os.chdir(codelists_path)
+    codelists.check()
+
+
+def test_codelists_check_passes_if_opencodelists_is_down(requests_mock, codelists_path):
+    requests_mock.post(
+        "https://www.opencodelists.org/api/v1/check/",
+        exc=requests.exceptions.ConnectionError,
+    )
     os.chdir(codelists_path)
     codelists.check()
```

Comment:
I appreciate that this isn't touched by your PR, but I think there should be an `assert` here, too.